tools/dotnet-msidentity/dotnet-msidentity.csproj

```diff
@@ -6,7 +6,6 @@
     <Nullable>enable</Nullable>
     <RootNamespace>Microsoft.DotNet.MSIdentity</RootNamespace>
     <SignAssembly>true</SignAssembly>
-    <LangVersion>preview</LangVersion>
```

Comment:
@deepchoudhery This is the one project that is being "downgraded" by this change, from `preview` to `latest`. Is there a need for it to be on preview currently that you're aware of?

Reply:
nope latest should be good now.